src/client.js

```diff
@@ -56,18 +56,13 @@ class Client extends EventEmitter {
   }
 
   callHttp (method, args, cb) {
-    get({
-      uri: this.uri + method,
-      qs: convertArgs(args),
-      json: true
-    }, (err, res, data) => {
-      if (err) return cb(err)
-      if (res.statusCode !== 200) {
-        let err = `Server responded with status code ${res.statusCode}`
-        return cb(Error(err))
-      }
-      if (data.error) return cb(Error(data.error))
-      cb(null, data.result)
+    axios({
+      url: this.uri + method,
+      params: args
+    }).then(({data}) => {
```

Comment:
Make sure you keep the `if (data.error) return cb(Error(data.error))` line here, since we might get a 200 response that tells us about an error.